packages/flutter_plugin_android_lifecycle/example/lib/main.dart

```diff
@@ -12,6 +12,9 @@ class MyApp extends StatelessWidget {
         appBar: AppBar(
           title: const Text('Sample flutter_plugin_android_lifecycle usage'),
         ),
+        body: Center(
+            child: Text(
+                'This plugin only provides Android Lifecycle API\n for other Android plugins. NO DART!')),
```

Comment:
I think this is a good message, but would probably be confused about the "NO DART!" part.

Could you reword this a bit, or even point the developer to a page on the Wiki / website?

Reply:
Ok I change the Text.